Dockerfile

```diff
@@ -103,6 +103,18 @@ RUN mkdir -p /opt && \
 ENV PATH $PATH:/opt/mips-mti-elf/2016.05-03/bin
 ENV MIPS_ELF_ROOT /opt/mips-mti-elf/2016.05-03
 
+# Install RISC-V binary toolchain
+RUN mkdir -p /opt && \
+        wget -q https://github.com/gnu-mcu-eclipse/riscv-none-gcc/releases/download/v7.2.0-2-20180110/gnu-mcu-eclipse-riscv-none-gcc-7.2.0-2-20180111-2230-centos64.tgz -O- \
+        | tar -C /opt -xz
+
+# HACK download arch linux' flex dynamic library
```

Comment:
I know this is ugly... Hopefully bionic beaver will solve the issue.
Or are there other options?

Reply:
Is there no RISC-V toolchain for Ubuntu?

Reply:
You mean, pre-packaged? Didn't find one.
The eclipse one used in this PR claims "tested on Debian, Ubuntu, ...". :/